### DISCUSSION TITLE
Syntax change in Woodpecker v1 workflows

### DISCUSSION THREAD

**Discussion by xoxys** [renovatebot/renovate#23568] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I am trying to get this working for the first time

### Describe the problem

Woodpecker v1 has changed the pipeline/workflow syntax. As a result, the woodpecker manager is not able to detect step images anymore. 

[v0.15.x ](https://woodpecker-ci.org/docs/0.15/usage/pipeline-syntax#image) syntax:

```
pipeline:
  backend:
    image: golang
``` 

[v1.0.x ](https://woodpecker-ci.org/docs/usage/pipeline-syntax#image) syntax:

```
steps:
  backend:
    image: golang
```

The woodpecker manager would need some adjustments to support both. 

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

### Have you created a minimal reproduction repository?

I have explained in the description why a minimal reproduction is impossible

**Comment by rarkins** [comment]
Please create a feature request discussion with reproduction repo plus links to relevant docs

**Reply by xoxys** [reply, marked as answer]
https://github.com/renovatebot/renovate/discussions/23569
